integreat_cms/static/src/js/translations/budget-graph.ts

```diff
@@ -0,0 +1,58 @@
+import { Chart, ChartConfiguration, DoughnutController, ArcElement } from "chart.js";
+
+Chart.register(DoughnutController, ArcElement);
+
+window.addEventListener("load", async () => {
+    const graph = document.querySelector("#budget-graph") as HTMLElement;
+    if (!graph) {
+        return;
+    }
+
+    // Display the correct renewal date for the MT budget
+    const renewalDateElement = document.querySelector("#deepl-renewal-date") as HTMLElement;
+    const renewalMonth = parseInt(renewalDateElement.dataset.renewalMonth, 10);
+    const now = new Date();
+    // Determine whether or not the next time the renewal month starts will happen this year or next year
+    const renewalDate = new Date(
+        Date.UTC(now.getFullYear() + (now.getMonth() < renewalMonth ? 0 : 1), renewalMonth, 1)
+    );
+    renewalDateElement.innerHTML += renewalDate.toLocaleDateString(undefined, {
```

Comment:
```suggestion
    renewalDateElement.textContent += renewalDate.toLocaleDateString(undefined, {
```
`textContent` is probably more future-proof against potential XSS vulnerabilities